### PR TITLE
Implement SensorML20EventObservation

### PR DESCRIPTION
**Pull request by SpeckiJ**

### CODE REVIEW

File: mappings/src/main/hbm/simple/dataset/DataResource.hbm.xml [hunk 1]
```diff
@@ -199,6 +199,13 @@
             </set>
         </subclass>
 
+        <subclass name="SensorML20DataEntity" extends="DataEntity" discriminator-value="sensorML20">
+            <property name="value" type="string">
```
Comment (CarstenHollmann) [inline mappings/src/main/hbm/simple/dataset/DataResource.hbm.xml:203]: use `text`

File: mappings/src/main/hbm/transactional/dataset/DataResource.hbm.xml [hunk 2]
```diff
@@ -293,6 +293,14 @@
             </set>
         </subclass>
 
+        <subclass name="SensorML20DataEntity" extends="DataEntity" discriminator-value="sensorML20">
+            <property name="value" type="string">
```
Comment (CarstenHollmann) [inline mappings/src/main/hbm/transactional/dataset/DataResource.hbm.xml:297]: use `text`

File: mappings/src/main/hbm/proxy/sampling/DataResource.hbm.xml [hunk 3]
```diff
@@ -300,6 +300,14 @@
             </set>
         </subclass>
 
+        <subclass name="SensorML20DataEntity" extends="DataEntity" discriminator-value="sensorML20">
+            <property name="value" type="string">
```
Comment (CarstenHollmann) [inline mappings/src/main/hbm/proxy/sampling/DataResource.hbm.xml:304]: use `text`

File: mappings/src/main/hbm/ereporting/dataset/DataResource.hbm.xml [hunk 4]
```diff
@@ -326,6 +326,14 @@
             </set>
         </subclass>
 
+        <subclass name="SensorML20DataEntity" extends="DataEntity" discriminator-value="sensorML20">
+            <property name="value" type="string">
```
Comment (CarstenHollmann) [inline mappings/src/main/hbm/ereporting/dataset/DataResource.hbm.xml:330]: use `text`

File: mappings/src/main/hbm/transactional/dataset/DataResource.hbm.xml (hunk 2 above)
Comment (SpeckiJ) [reply mappings/src/main/hbm/transactional/dataset/DataResource.hbm.xml]: refactored via: https://github.com/52North/sensorweb-server-db-model/pull/132/commits/09db0fd079bbebc376605c4ab951bc4c969b8149

File: mappings/src/main/hbm/simple/dataset/DataResource.hbm.xml (hunk 1 above)
Comment (SpeckiJ) [reply mappings/src/main/hbm/simple/dataset/DataResource.hbm.xml]: refactored via: https://github.com/52North/sensorweb-server-db-model/pull/132/commits/09db0fd079bbebc376605c4ab951bc4c969b8149

File: entities/src/main/java/org/n52/series/db/beans/parameter/ParameterFactory.java [hunk 5]
```diff
@@ -120,7 +120,13 @@ public static ParameterEntity<?> from(DescribableEntity entity, ValueType valueT
...
             entityType = EntityType.PLATFORM;
         }
-        return entityType != null ? from(entityType, valueType) : null;
+        if (entityType != null) {
+            ParameterEntity<?> e = from(entityType, valueType);
+            e.setDescribeableEntity(entity);
+            return e;
+        } else {
+            return null;
+        }
```
Comment (CarstenHollmann) [inline entities/src/main/java/org/n52/series/db/beans/parameter/ParameterFactory.java:129]: remove else and `return null;` directly

**Review by CarstenHollmann — CHANGES_REQUESTED** [review]
The definition is missing for some DataResource.hbm.xml files!

- ereporting/sampling
- proxy/dataset
- simple/sampling
- transactional/sampling

File: entities/src/main/java/org/n52/series/db/beans/parameter/ParameterFactory.java (hunk 5 above)
Comment (SpeckiJ) [reply entities/src/main/java/org/n52/series/db/beans/parameter/ParameterFactory.java:127]: implemented in https://github.com/52North/sensorweb-server-db-model/pull/132/commits/44b891c1002474122a6ea1801cd222d51080310d

**Review by CarstenHollmann — APPROVED** [review]

**Pull request merged**